src/tls/openssl.c

```diff
@@ -426,6 +437,15 @@ static int tls_net_handshake(struct flb_tls *tls, void *ptr_session)
         if (ret != SSL_ERROR_WANT_READ &&
             ret != SSL_ERROR_WANT_WRITE) {
             ret = SSL_get_error(session->ssl, ret);
+            // The SSL_ERROR_SYSCALL with errno value of 0 
```

Comment:
https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#multiline-comment
Multiline comment should be
```
/*
 * This is my comment which is longer than 80 characters, so we must use the
 * multi-line type comments.
 */
```